scipy/stats/_continuous_distns.py

```diff
@@ -10085,7 +10085,7 @@ def solve_for_kappa(kappa):
                     return sc.i1e(kappa)/sc.i0e(kappa) - r
 
                 root_res = root_scalar(solve_for_kappa, method="brentq",
-                                    bracket=(1e-8, 1e12))
+                                       bracket=(np.finfo(float).tiny, 1e16))
```

Comment:
I don't mind this, but why is 1e16 enough if 1e12 isn't?

This will tend to add a small number of iterations when  the shape is a moderate value.

I think I will add a private bracket finder helper function to avoid the need for fixed bracket here. (The logic of expanding an initial bracket with a loop is pretty short, and you can find it all over stats, but it comes up so often we should have a function.)

Reply:
`1e12` was set arbitrarily. `1e16` is the maximum value that can still be fitted before `i1e(kappa)/i0e(kappa)=1`. I think we should always try to make the fit work well over a wide range of values, even exotic ones.

That bracket finder sounds very helpful indeed!